scripts/o3de/ExportScripts/export_standalone_monolithic_project_centric.py

```diff
@@ -281,15 +315,21 @@ def export_standalone_monolithic_project_centric(project_path: pathlib.Path,
                                             preferred_asset_list_path,
                                             max_bundle_size)
     else:
-        expected_bundles_path = project_path / "AssetBundling/Bundles"    
-    for should_do, expected_output_dir, project_file_patterns in zip(
+        expected_bundles_path = project_path / "AssetBundling/Bundles"
+    zip_pattern_game_launcher = [f'{project_name}.GameLauncher{EXECUTABLE_EXTENSION}']
+    zip_pattern_server_launcher = [f'{project_name}.ServerLauncher{EXECUTABLE_EXTENSION}', f'{project_name}.ServerLauncher{EXECUTABLE_EXTENSION}']
```

Comment:
Uh, :ServerLauncher" is repeated instead of "HeadlessServerLauncher"
```suggestion
    zip_pattern_server_launcher = [f'{project_name}.ServerLauncher{EXECUTABLE_EXTENSION}', f'{project_name}.HeadlessServerLauncher{EXECUTABLE_EXTENSION}']
```